packages/mobile/src/components/core/FlatList.tsx

```diff
@@ -34,6 +39,16 @@ const CollapsibleFlatList = (props: CollapsibleFlatListProps) => {
           scrollPropsAndRef.onScroll,
           props.onScroll
         )}
+        refreshControl={
+          Platform.OS === 'ios' ? undefined : (
+            <RefreshControl
+              progressViewOffset={scrollPropsAndRef.progressViewOffset}
+              refreshing={!!refreshing}
+              onRefresh={onRefresh ?? undefined}
```

Comment:
i believe as long as onRefresh is undefined coming down don't need the ??. Is it null or something sometimes?

Reply:
Yeah it has a `null` in there so typescript was angry